src/Symfony/Component/Notifier/Bridge/LineBot/Tests/LineBotTransportTest.php

```diff
@@ -66,4 +117,41 @@ public function testSendWithErrorResponseThrows()
 
         $transport->send(new ChatMessage('testMessage'));
     }
+
+    /**
+     * @dataProvider validMessageProvider
+     */
+    public function testValidMessage(LineBotOptions $lineBotOptions, ChatMessage $message)
+    {
+        $client = $this->createMock(HttpClientInterface::class);
+
+        $client->expects($this->once())
+            ->method('request')
+            ->willThrowException(new \Exception('The request should not be sent'))
+            ->with(
+                $this->anything(),
+                $this->anything(),
+                $this->equalTo([
+                    'auth_bearer' => 'testToken',
+                    'json' => $lineBotOptions->toArray(),
+                ]),
+            )
+        ;
+
+        $this->expectException(\Exception::class);
+
+        $transport = $this->createTransport($client);
+        $transport->send($message);
+    }
+
+    public function testSendNotificationWithInvalidOptions()
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('Invalid message provided.');
+
+        $transport = $this->createTransport();
```

Comment:
```suggestion
        $transport = $this->createTransport();
        
        $this->expectException(InvalidArgumentException::class);
        $this->expectExceptionMessage('Invalid message provided.');        
```
Lets move it as close to the throwing point as possible, thanks